### ISSUE TITLE
add a prominent "edit" button that links to Wikidata

### ISSUE THREAD

**Issue by egonw** [fnielsen/scholia#802]
Encouraging users to edit the content.

**Comment by fnielsen** [comment]
Yes. I wonder if you can give an example. I see a problem if there is a missing work for a specific person. Then the button should not go to the Q for the person but rather to new item or a DOI entry field.

**Comment by Daniel-Mietchen** [comment]
Perhaps we should be using TABernacle for that, as per #516 .
